components/slack/actions/send-block-kit-message/send-block-kit-message.mjs

```diff
@@ -0,0 +1,43 @@
+import buildBlocks from "../common/build-blocks.mjs";
+import common from "../common/send-message.mjs";
+
+export default {
+  ...common,
+  ...buildBlocks,
+  key: "slack-send-block-kit-message",
+  name: "Build and Send a Block Kit Message",
+  description: "Configure custom blocks and send to a channel, group, or user. [See the documentation](https://api.slack.com/tools/block-kit-builder).",
+  version: "0.4.0",
```

Comment:
Should probably be `0.0.1` if it's new

Reply:
It's an action that existed previously, but was removed. I've encountered issues with republishing components at version `0.0.1` if they were previously published at a higher version. Since this one was at `0.3.3` when it was removed, I decided to use `0.4.0`.

Reply:
Gotcha!